lib/Consul/API/Agent.pm

```diff
@@ -77,6 +81,10 @@ sub force_leave {
 sub check_register {
     my ($self, $check, %args) = @_;
     croak 'usage: $agent->check_register($check, [%args])' if grep { !defined } ($check);
+    {
+        local $Carp::CarpInternal{ (__PACKAGE__) } = 1;
```

Comment:
Fair enough. `Carp::Clan` is my go-to for that sort of thing. This is fine though!